pyproject.toml

```diff
@@ -37,6 +37,7 @@ skip_glob = '''
 [tool.pylint]
 max-line-length = 120
 ignore = ['.eggs', '.git', '.tox', '.venv', '.build', 'report', 'tests']
+disable = ['C0204']
```

Comment:
Why do we need to disable this check? Are we even using metaclasses (and their class methods)? [ref](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/bad-mcs-classmethod-argument.html)

Reply:
yeah pylint does not like what we are doing in https://github.com/canonical/prometheus-juju-exporter/blob/e89e1e52621ba7e15d784f45a8106fcc84b3cd96/prometheus_juju_exporter/config.py#L15

upon renaming, pflake8 does not like it, due to N804: first argument of a classmethod should be named ‘cls’

i picked one of them

Reply:
Yea, this is caused by a recent pylint change in 2.16.x. I too agree to disable "C0204" because it seems to violates PEP8. There's a discussion in pylint repo about whether to disable this check by default or revert the change (https://github.com/PyCQA/pylint/issues/8162)

Reply:
Thanks, yeah disabling it is probably a way to go.